scripts/migration/v0.0.5__add-space-slug.ts

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node -r esm -r ts-node/register
+
+import { MigrateOptions } from "fireway";
+
+import { generateSlug } from "../../functions/src/utils/venue";
+
+export const migrate = async ({ firestore }: MigrateOptions) => {
+  console.log(`Fetching spaces...`);
+
+  const spaces = await firestore.collection("venues").get();
+
+  console.log(`Updating spaces...`);
+
+  try {
+    for (const space of spaces.docs) {
```

Comment:
nit:
```suggestion
  const { docs: spaces } = await firestore.collection("venues").get();

  console.log(`Updating spaces...`);

  try {
    for (const space of spaces) {
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2584/commits/951fc2b03c6f9ae842867d9473c64c2d759ed03f